R/replay-html.r

```diff
@@ -113,6 +113,29 @@ replay_html.recordedplot <- function(x, name_prefix, obj_id, ...) {
   paste0("<img src='", escape_html(path), "' alt='' width='540' height='400' />")
```

Comment:
on an unrelated note, this fixed aspect ratio can make the plot look a bit funny.